### PR TITLE
Backport long long bugfix

### PR DESCRIPTION
**Pull request by xantares**

### CODE REVIEW

File: recipe/meta.yaml
Comment (isuruf) [inline recipe/meta.yaml]: I wonder whether a build number bump is sufficient for an API change.

Comment (isuruf) [reply recipe/meta.yaml]: @xantares, can you ask upstream for a new release? or change the version to `2.7.3.dev0` or something like that? cc @conda-forge/core

Comment (xantares) [reply recipe/meta.yaml]: see what upstream says https://github.com/wbhart/mpir/issues/182

Comment (isuruf) [reply recipe/meta.yaml]: Looks like it will take some time for the release. I'd be happy to merge this PR if the version is set to `2.7.3.dev0`

File: recipe/meta.yaml
Comment (isuruf) [inline recipe/meta.yaml:1]: :+1:

File: recipe/meta.yaml
Comment (isuruf) [reply recipe/meta.yaml]: Can you change this to 0 now?

**Review by isuruf — APPROVED** [review]

**Comment by isuruf** [comment]
@xantares, let me know if you'd like to be added as a maintainer.

**Pull request merged**
